scripts/sync-versions.mjs

```diff
@@ -37,7 +37,8 @@ function getProjectVersion() {
 }
 
 function createReleaseCommit(version) {
+  const message = `chore(release): ${version}'`;
```

Comment:
I think you need to remove the single quote at the end
`chore(release): ${version}'` -> `chore(release): ${version}`

Reply:
Copypasta fail! 🤦  Thanks, Katie!